cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -443,7 +448,17 @@ def create_cephCSIKeyring_cephFSProvisioner(self):
         '''
         command: ceph auth get-or-create client.csi-cephfs-provisioner mon 'allow r' mgr 'allow rw' osd 'allow rw tag cephfs metadata=*'
         '''
-        cmd_json = {"prefix": "auth get-or-create",
+        metadata_pool = self._arg_parser.cephfs_metadata_pool_name
```

Comment:
What if `cluster_name` is empty? We should error out

Reply:
Yes we should raise an error